docs/web/migrating-to-v2.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Migrating to v2
+sidebar_position: 0
+---
+
+Version 2 of Connect-ES is available. They key changes are:
```

Comment:
```suggestion
Version 2 of Connect-ES is available. The key changes are:
```

Reply:
Thanks, fixed in https://github.com/connectrpc/connectrpc.com/pull/226/commits/329f28a9af4c08cb7d20066d13ddf61cd14f5e27.